peps/tests/test_commands.py

```diff
@@ -6,29 +7,37 @@
 from django.conf import settings
 from django.core import serializers
 from django.core.management import call_command
-from django.core.exceptions import ImproperlyConfigured
 
 import responses
 
 from pages.models import Image
 
-from . import FAKE_PEP_REPO
+from . import FAKE_PEP_ARTIFACT
 
 
-@responses.activate
 class PEPManagementCommandTests(TestCase):
 
-    def setUp(self):
-        self.peps_tar = io.open(FAKE_PEP_ARTIFACT, 'rb')
+    @classmethod
+    def setUpTestData(cls):
+        with io.open(FAKE_PEP_ARTIFACT, 'rb') as fp:
+            peps_tar = fp.read()
```

Comment:
We need to reuse peps_tar in each test, so change to use bytes instead of file-like object.